arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1229,14 +1257,14 @@
       }
     },
     "OperationsMonitoringProperties": {
-      "description": "The Operation Monitoring properties.",
+      "description": "The operations monitoring properties for the IoT hub. The possible keys to the dictionary are Connections, DeviceTelemetry, C2DCommands, DeviceIdentityOperations, FileUploadOperations.",
       "type": "object",
       "properties": {
         "events": {
           "type": "object",
           "additionalProperties": {
             "enum": [ "None", "Error", "Information" ],
```

Comment:
You are right, I had fixed this a while back in the azure-resource-manager-schemas repo.. Fixed this in the swagger now. Thanks for reporting.